parlai/tasks/eli5/data_creation/data_utils.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Taken from
+https://github.com/facebookresearch/ELI5/blob/master/data_creation/data_utils.py.
+"""
+
+import json
+import math
+import re
+import signal
+import subprocess
+
+from contextlib import contextmanager
+from glob import glob
+from os.path import join as pjoin
+from os.path import isfile
+from os.path import isdir
+
+from spacy.lang.en import English
```

Comment:
do we need to add `spacy` to `requirements.txt` at the top level?

Reply:
Not at the top level. We should wrap it in a try/except here saying to go to spacy.io and follow installation instructions.